proposals/lip-0054.md

```diff
@@ -63,13 +63,13 @@ Similar to the case for message recovery commands, it is not guaranteed to recov
 
 In summary, the functionality provided by these recovery commands applies for sidechains that were terminated for inactivity or [other violations of the interoperability protocol][lip-0053#violations]. If the validators of the terminated sidechain were byzantine in the past, i.e., the security guarantees of the sidechain were broken, it is likely that these recovery mechanisms would not work.
 
-### Message Recovery from the Sidechain Account Outbox
+### Message Recovery from the Sidechain Channel Outbox
 
-This mechanism allows to recover any CCM pending in the sidechain outbox. That is, those CCMs that have not been included in the receiving sidechain yet.
-Specifically, this includes all the CCMs whose indices (the position of these CCMs in the sidechain outbox tree) are larger than the index of the last message that the receiving sidechain reported to have included in its inbox on the mainchain. This recovery mechanism requires these conditions to work as follows:
+This mechanism allows to recover any CCM pending in the sidechain channel outbox. That is, those CCMs that have not been included in the receiving chain yet.
```

Comment:
Could you please specifically say here which way the CCM is going? This created some confusion for me.

Reply:
edited